### PR TITLE
Fix endurance and journalspy settings to work with current systemd

### PR DESCRIPTION
**Pull request by pvuorela**
@saukko @rainemak

### CODE REVIEW

**Comment by pvuorela** [comment]
Yes, could use systemd d-bus interface too, but thought this would be good enough for now.

File: rpm/crash-reporter.spec [hunk 1]
```diff
@@ -116,13 +116,6 @@ make %{_smp_mflags}
 /usr/lib/pkgconfig/*.pc
 
 %post
-# Remove timer unit symlink potentially left by old installation.
-rm -f /etc/systemd/system/basic.target.wants/crash-reporter-endurance.timer
```
Comment (rainemak) [inline rpm/crash-reporter.spec]: When this did exist?

File: src/sailfishui/crash-reporter.qml [hunk 2]
```diff
@@ -111,35 +110,33 @@ Page {
...
                 text: qsTrId("settings_crash-reporter_enable_endurance")
                 //% "Reports system statistics helping diagnose problems that "
                 //% "manifest themselves only after a long-term use of the "
                 //% "device like memory leaks, excessive battery drain, or "
                 //% "decreasing performance."
                 description: qsTrId("settings_crash-reporter_enable_endurance_description")
-
-                onBeforeStateChange: {
-                    PrivacySettings.endurance = newState
+                automaticCheck: false
```
Comment (rainemak) [inline src/sailfishui/crash-reporter.qml]: Initial state is broken now (same for Journal Spy)

Add...
```
checked: PrivacySettings.endurance
```

File: rpm/crash-reporter.spec (hunk 1 above)
Comment (pvuorela) [reply rpm/crash-reporter.spec]: These removals came on 2014-04 and 2014-02. Should be safe by now.

File: src/sailfishui/crash-reporter.qml (hunk 2 above)
Comment (pvuorela) [reply src/sailfishui/crash-reporter.qml]: doh :)

**Comment by pvuorela** [comment]
Fixed initial state.

File: rpm/crash-reporter.spec (hunk 1 above)
Comment (rainemak) [reply rpm/crash-reporter.spec]: Ok, thanks for clarification.

**Comment by rainemak** [comment]
LGTM :+1:

**Pull request merged**